lib/api/v3/work_packages/schema/work_package_schema_representer.rb

```diff
@@ -0,0 +1,186 @@
+                   getter: -> (*) {
+                     link = api_v3_paths.available_responsibles(represented.project.id)
+                     representer = ::API::Decorators::AllowedValuesByLinkRepresenter.new(
+                       type: 'User',
+                       name: I18n.t('activerecord.attributes.work_package.responsible'))
+                     representer.required = false
+
+                     if represented.defines_assignable_values?
+                       representer.allowed_values_href = link
+                     end
+
+                     representer
+                   }
+
+          property :status,
+                   exec_context: :decorator,
+                   getter: -> (*) {
+                     assignable_statuses = represented.assignable_statuses_for(current_user)
+                     representer = ::API::Decorators::AllowedValuesByCollectionRepresenter.new(
+                       type: 'Status',
+                       name: I18n.t('activerecord.attributes.work_package.status'),
+                       current_user: current_user,
+                       value_representer: API::V3::Statuses::StatusRepresenter,
+                       link_factory: -> (status) do
+                           { href: api_v3_paths.status(status.id), title: status.name }
+                         end)
+
+                     representer.allowed_values = assignable_statuses
+                     representer
+                   }
+
+          property :version,
+                   exec_context: :decorator,
+                   getter: -> (*) {
+                     representer = ::API::Decorators::AllowedValuesByCollectionRepresenter.new(
+                       type: 'Version',
+                       name: I18n.t('activerecord.attributes.work_package.fixed_version'),
+                       current_user: current_user,
+                       value_representer: API::V3::Versions::VersionRepresenter,
+                       link_factory: -> (version) do
+                           { href: api_v3_paths.version(version.id), title: version.name }
+                       end)
+
+                     representer.required = false
+                     representer.allowed_values = represented.assignable_versions
+                     representer
+                   }
+
+          property :priority,
+                   exec_context: :decorator,
+                   getter: -> (*) {
+                     representer = ::API::Decorators::AllowedValuesByCollectionRepresenter.new(
+                       type: 'Priority',
+                       name: I18n.t('activerecord.attributes.work_package.priority'),
+                       current_user: current_user,
+                       value_representer: API::V3::Priorities::PriorityRepresenter,
+                       link_factory: -> (priority) do
+                           { href: api_v3_paths.priority(priority.id), title: priority.name }
+                         end)
```

Comment:
`end` at 169, 25 is not aligned with `-> (priority) do` at 167, 37 or `link_factory: -> (priority) do` at 167, 23